agents-api/README.md

```diff
@@ -1,48 +1,42 @@
+   - **Entities**: Agents, developers, documents, entries, memories, sessions, etc.
 
-The `models` module encapsulates all data interactions with the CozoDB database, providing a structured way to perform CRUD operations and other specific data manipulations across various entities.
+### 2. **Routers**
+   - **Purpose**: Handles HTTP requests and directs them to the appropriate handler functions for operations related to agents, documents, etc.
 
-### Routers
+### 3. **Dependencies**
+   - **Purpose**: Houses essential components such as authentication, developer identification, and custom exception handling required for the API to function.
 
-The `routers` module handles HTTP routing for different parts of the application, directing incoming HTTP requests to the appropriate handler functions.
+### 4. **Common Utilities**
+   - **Purpose**: Provides utility functions for interacting with the Cozo API client, handling dates, time, and custom JSON operations.
 
-### Dependencies
+### 5. **Activities**
+   - **Purpose**: Facilitates memory management, relationship summarization, and generating insights from dialogues with agents.
 
-This module contains components crucial for the operation of the agents-api, including authentication, developer identification, and custom exception handling.
+### 6. **Clients**
+   - **Purpose**: Contains classes and functions to interface with external services and APIs, abstracting external service details from the core application logic.
 
-### Common Utilities
+### 7. **Workers**
+   - **Purpose**: Manages background tasks and enhances performance by offloading heavy tasks such as agent task executions.
 
-The `utils` module offers a collection of utility functions designed to support various aspects of the application, including interactions with the Cozo API client, date and time operations, and custom JSON utilities.
 
-### Activities
+## Getting Started with Development
 
-The `activities` module facilitates various activities related to agent interactions, such as memory management, generating insights from dialogues, and summarizing relationships.
-
-### Clients
-
-The `clients` module contains client classes and functions for interacting with various external services and APIs, providing a simplified interface for the rest of the application.
-
-### Workers
-
-This module is responsible for handling background tasks and jobs for the Agents API application, enhancing the application's performance and scalability.
-
-## Getting Started
-
-To set up the project for development:
-
-1. Clone the repository.
-2. Install dependencies.
-3. Run the application.
-
-This guide provides a brief overview for new contributors to get started with the project development.
+1. **Clone the Repository**:
+   - Retrieve the source code to your local environment.
+3. **Install Dependencies**:
+   - Run `npm install` or the relevant package manager to install the necessary modules.
```

Comment:
Could you please elaborate on the reason for using `npm install` for this project?

Reply:
yes this is incorrect. we use poetry, so it'd be `poetry install` instead.